src/modules/git_commit.rs

```diff
@@ -201,6 +204,42 @@ mod tests {
         repo_dir.close()
     }
 
+    #[test]
+    fn test_render_commit_hash_with_tag_disabled() -> io::Result<()> {
+        let repo_dir = fixture_repo(FixtureProvider::Git)?;
+
+        let mut git_commit = create_command("git")?
+            .args(&["rev-parse", "HEAD"])
+            .current_dir(&repo_dir.path())
+            .output()?
+            .stdout;
+        git_commit.truncate(7);
+        let commit_output = str::from_utf8(&git_commit).unwrap().trim();
+
+        create_command("git")?
+            .args(&["describe", "--tags", "--exact-match", "HEAD"])
+            .current_dir(&repo_dir.path());
+
+        let actual = ModuleRenderer::new("git_commit")
+            .config(toml::toml! {
+                [git_commit]
+                    only_detached = false
+                    tag_symbol = ""
```

Comment:
`tag_disabled` does not default to `true`: 
```suggestion
                    tag_symbol = ""
                    tag_disabled=true
```

Reply:
I think it it default to `true`:
https://github.com/starship/starship/blob/c3cd499a30ccefcc14eaa60156da7ef98ab6390c/src/configs/git_commit.rs#L31

Reply:
I think I would prefer changing the default to `false`, either way this is something of a breaking change from v0.10. Any thoughts for/against @starship/astronauts?

Reply:
I concur that changing it to false would probably be better. I do think we should probably merge this now as a bugfix and change the default with the 2.0.0 release.

Reply:
Ok. Let's revisit this later then.